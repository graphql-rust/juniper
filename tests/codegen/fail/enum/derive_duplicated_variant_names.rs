#[derive(juniper::GraphQLEnum)]
enum Test {
    Test,
    #[graphql(name = "TEST")]
    Test1,
}

fn main() {}
