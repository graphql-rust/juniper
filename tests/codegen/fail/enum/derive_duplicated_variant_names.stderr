error: GraphQL enum expected all enum variants to have unique names
 --> fail/enum/derive_duplicated_variant_names.rs:3:5
  |
3 | /     Test,
4 | |     #[graphql(name = "TEST")]
5 | |     Test1,
  | |__________^
